test/regressions/index.js

```diff
@@ -21,6 +21,7 @@ const blacklist = [
   'docs-components-data-grid-filtering/ColumnTypeFilteringGrid.png', // Needs interaction
   'docs-components-data-grid-filtering/CustomRatingOperator.png', // Needs interaction
   'docs-components-data-grid-filtering/ExtendNumericOperator.png', // Needs interaction
+  'docs-components-data-grid-components/CustomFooter.png', // Needs @material-ui/lab v4
```

Comment:
Could you detail the issue here?

Reply:
I upgraded all @material-ui/xxx dependencies in the root's package.json to v5, so this regression test will now run with @material-ui/lab v5. However, the Rating component in v5 is not in the lab anymore, then it throws an "The Rating component was moved from the lab to the core" error in the console. Since no test can output to the console it fails.

https://github.com/mui-org/material-ui-x/blob/e214b3a43f639980fde82ff9d62eb40612aa335f/test/regressions/index.test.js#L107

Reply:
OK, so maybe we should do
```suggestion
  // TODO import the Rating from @mui/material, not the lab.
  'docs-components-data-grid-components/CustomFooter.png',
```